builtin/game/deprecated.lua

```diff
@@ -70,3 +70,17 @@ core.setting_get = setting_proxy("get")
 core.setting_setbool = setting_proxy("set_bool")
 core.setting_getbool = setting_proxy("get_bool")
 core.setting_save = setting_proxy("write")
+
+--
+-- core.register_on_auth_fail
+--
+
+core.register_on_auth_fail = function (func)
```

Comment:
This notation is mostly used in builtin:
`function core.register_on_auth_fail(func)`